charts/kube-prometheus-stack/values.yaml

```diff
@@ -3323,6 +3323,6 @@ thanosRuler:
   #     foo:$apr1$OFG3Xybp$ckL0FHDAkoXYIlH9.cysT0
   #     someoneelse:$apr1$DMZX2Z4q$6SbQIfyuLQd.xmo/P0m2c.
 
-## Setting to true produces cleaner names, but may require a data migration because of persistent volume claim name change.
+## Setting to true produces cleaner resource names, but requires a data migration because the name of the persistent volume changes. Therefore this should only be set once on inital installation.
```

Comment:
sorry, typo.

```suggestion
## Setting to true produces cleaner resource names, but requires a data migration because the name of the persistent volume changes. Therefore this should only be set once on initial installation.
```